src/screens/WebsocketResearchScreen/SampleChatScreen.tsx

```diff
@@ -4,75 +4,87 @@
 import * as React from 'react';
 import {Dimensions, FlatList, KeyboardAvoidingView, Platform, StyleSheet, View} from 'react-native';
 
-import AnonymousChatHeader from '../../components/AnonymousChat/AnonymousChatHeader';
 import AnonymousInputMessage from '../../components/Chat/AnonymousInputMessage';
 import BaseChatItem from '../../components/AnonymousChat/BaseChatItem';
-import useAnonymousChatScreenHook from '../../hooks/screen/useAnonymousChatScreenHook';
+import ChatDetailHeader from '../../components/AnonymousChat/ChatDetailHeader';
+import useChatScreenHook from '../../hooks/screen/useChatScreenHook';
+import {ANONYMOUS} from '../../hooks/core/constant';
 import {colors} from '../../utils/colors';
 
 const {height} = Dimensions.get('window');
 
+export const styles = StyleSheet.create({
+  keyboardAvoidingView: {
+    flex: 1,
+    backgroundColor: colors.white
+  },
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    height: height - 85
+  },
+  chatContainer: {
+    display: 'flex',
+    height: '100%',
+    marginBottom: 72
+  },
+  inputContainer: {
+    backgroundColor: colors.white,
+    position: 'absolute',
+    bottom: 0,
+    // height: 50,
+    left: 0,
+    right: 0,
+    zIndex: 100,
+    padding: 8,
+    paddingBottom: 16,
+    borderTopColor: colors.lightgrey,
+    borderTopWidth: 1
+  }
+});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles have been moved outside of the `SampleChatScreen` component. This is a good change as it improves performance by preventing the styles from being recreated on every render.

```diff
- const styles = StyleSheet.create({
+ export const styles = StyleSheet.create({
  // ...
});
```

Overall, these changes improve the flexibility and adaptability of the code to handle different types of chats. The use of constants and additional props allows for more customization and specificity in the behavior of the components and hooks.


<!-- This is an auto-generated comment by OSS CodeRabbit -->